addons/account/static/src/components/x2many_buttons/x2many_buttons.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="account.X2ManyButtons">
+        <t t-foreach="this.current_field.records" t-as="record" t-key="record.id" class="d-inline">
+            <t t-if="record_index lt 3">
+                    <button class="btn btn-link p-0 ms-1" t-on-click="() => this.openFormAndDiscard(record.resId)" t-att-data-hotkey="`shift+${record_index + 1}`"><t t-out="record.data.display_name"/></button>
+                    <span t-if="record_index lt loop_record_count - 1">,</span>
```

Comment:
I think you forgot to fix the indentation here. :grin:

(There are 8 more spaces instead of just 4 like all the other lines of this file)